src/align.rs

```diff
@@ -141,6 +141,15 @@ pub struct AlignedVec<T: Copy, C: AlignedByteChunk> {
 }
 
 impl<T: Copy, C: AlignedByteChunk> AlignedVec<T, C> {
+    // Note that in Rust, no single allocation can exceed `isize::MAX` __bytes_.
```

Comment:
What is the `__` and `_` surrounding `bytes`?  Is that supposed to be Markdown?

Reply:
That was meant to underscore that this is a size in terms of bytes, not units. Apparently that emphasis didn't achieve its goal. I can just remove it?

Reply:
```suggestion
    // Note that in Rust, no single allocation can exceed `isize::MAX` bytes.
```

Reply:
If that's supposed to be markdown for bold, that's fine, I was just confused by the 2 underscores before and only 1 after.  Could you just do 1 before and after? And also for the other place you did the same thing.